docs/2.8.0/docs/canton/usermanual/kms/external_key_storage_aws.rst

```diff
@@ -0,0 +1,199 @@
+..
+   Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates.
+..
+   Proprietary code. All rights reserved.
+
+.. _external_key_storage_aws:
+
+Configure External Key Storage and Usage with a AWS KMS
+=======================================================
+
+.. enterprise-only::
+
+The following section describes the steps needed to enable :ref:`External Key Storage and Usage <kms_external_architecture>`
+in Canton using AWS KMS. These steps include configuring AWS KMS, as well as, configuring this particular mode of operation.
+
+.. note::
+    Nodes configured with and without external KMS key storage can interact with each other,
+    including nodes using different KMS providers, as long as they follow the same crypto schemes.
```

Comment:
This is true but `as long as they follow the same crypto schemes.` is a big enough caveat that we should emphasize it more. At least link it to the "Interoperability" section (see my other comment below)